packages/browser-integration-tests/suites/replay/bufferMode/test.ts

```diff
@@ -325,7 +325,7 @@ sentryTest(
     expect(
       await page.evaluate(() => {
         const replayIntegration = (window as unknown as Window & { Replay: InstanceType<typeof Replay> }).Replay;
-        const replay = replayIntegration['_replay'];
+        const replay = replayIntegration._replay;
```

Comment:
we did this to get around typescript error - doesn't apply anymore?

Reply:
This was an `unsafe` fix. I reverted the changes, and used only safe fixes. There are still some errors, but they're only on the linting side. I'll check back this once the rest is done.